README.md

```diff
@@ -1,30 +1,91 @@
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
 [![][travis img]][travis]
 [![][maven img]][maven]
 [![][release img]][release]
 [![][license-epl img]][license-epl]
 [![][license-edl img]][license-edl]
 
-Eclipse Collections is a collections framework for Java. It has optimized List, Set and Map implementations with a rich and fluent API.  The library provides additional data structures not found in the JDK like Bags, Multimaps and BiMaps.  The framework also provides primitive versions of Lists, Sets, Bags, Stacks and Maps with a rich and fluent API.  There is support for both Mutable and Immutable versions of all containers in the library.  The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
+![logo](artwork/eclipse-collections-logo.png) 
+
+
+## [Eclipse Collections](http://www.eclipse.org/collections)
+##### [中文](https://www.eclipse.org/collections/cn/index.html), [Deutsch](https://www.eclipse.org/collections/de/index.html), [Français](https://www.eclipse.org/collections/fr/index.html), [日本語](https://www.eclipse.org/collections/ja/index.html), [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html), [Русский](https://www.eclipse.org/collections/ru/index.html)
+Eclipse Collections is a collections library for Java with a rich, functional and fluent API. The library provides optimized List, Set and Map implementations and has data structures not found in the JDK including Bags, Multimaps and BiMaps. There is support for primitive versions of Lists, Sets, Bags, Stacks and Maps for all primitive types. There are Mutable and Immutable versions of all container types. The library has support for eager, lazy, and parallel iteration patterns, with eager APIs available directly on the containers. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
+
+
+## Learn Eclipse Collections
+
+* The [Eclipse Collections Katas](https://github.com/eclipse/eclipse-collections-kata), a fun way to help you learn idiomatic Eclipse Collections usage.
+* The [Eclipse Collections Reference Guide](https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md), an overview of the extensive features available in the framework.
+* The [Javadoc](https://www.eclipse.org/collections/javadoc/9.2.0/overview-summary.html) 
+* [API Design of Eclipse Collections](https://dzone.com/articles/jvm-advent-calendar-api-design-of-eclipse-collecti)
+* [Ten Reasons to use Eclipse Collections](https://medium.com/oracledevs/ten-reasons-to-use-eclipse-collections-91593104af9d)
+
+
+## How to Contribute
+
+We welcome contributions! We accept contributions via pull requests here in GitHub. Please see [How To Contribute](CONTRIBUTING.md) to get started.
```

Comment:
The 99% use-case is using the library without contributing. This belongs at the end.

Reply:
Moved